src/frontend/src/components/ChangeRequestRow.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Typography, Grid, useTheme } from '@mui/material';
+import { Box } from '@mui/system';
+import { ChangeRequest, ChangeRequestRowInfo } from 'shared/src/types/change-request-types';
+import ChangeRequestDetailCard from './ChangeRequestDetailCard';
+
```

Comment:
instead of defining the interface in the change request types define it as a prop and pass it in kinda like this example

interface ProjectViewContainerProps {
  project: Project;
  enterEditMode: () => void;
}

const ProjectViewContainer: React.FC<ProjectViewContainerProps> = ({ project, enterEditMode }) => {


Also see comment below for further clarification